inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongGroupController.java

```diff
@@ -195,4 +196,12 @@ public Response<Boolean> reset(@RequestBody @Validated InlongGroupResetRequest r
         String operator = LoginUserUtils.getLoginUser().getName();
         return Response.success(groupProcessOperation.resetGroupStatus(request, operator));
     }
+
+    @RequestMapping(value = "/group/detail/{groupId}", method = RequestMethod.GET)
+    @ApiOperation(value = "get group detail")
+    @ApiImplicitParam(name = "groupId", value = "Inlong group id", dataTypeClass = String.class, required = true)
+    public Response<Map<String, Object>> detail(@PathVariable String groupId) {
```

Comment:
It is very strongly recommended not to return a Map in the REST interface because the data in the Map is not intuitive, it is equivalent to a black box, and it is very unfriendly to positioning problems.

The returned data should be abstracted as a POJO, please refer to other interfaces for modification, thank you.